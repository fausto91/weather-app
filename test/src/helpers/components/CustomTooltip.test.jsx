const { render } = require("@testing-library/react");
const { CustomTooltip } = require("../../../../src/helpers/components/CustomTooltip");

describe('Prueba en componente CustomTooltip', () => { 
    const text = "Temperatura actual"
    test('Deberia de hacer match con el snapshot', () => { 
        const {container} = render( <CustomTooltip text={text}></CustomTooltip>);
        expect(container).toMatchSnapshot();
     });

 });