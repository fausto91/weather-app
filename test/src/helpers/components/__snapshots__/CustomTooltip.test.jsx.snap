// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba en componente CustomTooltip Deberia de hacer match con el snapshot 1`] = `
<div>
  <span
    aria-label="Temperatura actual"
    class=""
    data-mui-internal-clone-element="true"
  />
</div>
`;
